client/src/App.js

```diff
@@ -118,6 +120,26 @@ const App = props => {
         data: { refreshr_id: res.data.newRefreshrID }
       })
         .then(res => {
+          // console.log(res.data.message)
+//           setMessage(res.data.message);
+//         })
+//         .catch(err => {
+//           console.log(err);
+//         });
+//     });
+//   };
+
+//   //add questions
+//   const addQuestions = question => {
+//     //console.log('Question from addQuestions ===', question);
+//     axios({
+//       method: 'post',
+//       //Development
+//       url: 'http://localhost:9000/questions',
+//       //Production
+//       //url: 'https://refreshr.herokuapp.com/questions',
+//       headers: { Authorization: `Bearer ${token}` },
+//       data: question
```

Comment:
Can you get rid of these commented out lines, I'm worried it will create a merge conflict

Reply:
Yeah I did that because there were merge conflicts and I wasn't 100% sure which code to choose. So I commented out the one that seemed less likely to be used. What do you think is a better way of doing this?
Perhaps resolving merge conflicts from the terminal rather than GitHub so I can run the changes locally before a PR?
I've removed the commented out code for this one 👍

Reply:
Hmm weird,  I think if you pull before pushing, then you can always choose the code coming from the branch where you pulled from